core/trino-main/src/main/java/io/trino/server/SliceSerialization.java

```diff
@@ -20,10 +20,11 @@
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
 import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
 
 import java.io.IOException;
 
-import static io.airlift.slice.Slices.utf8Slice;
+import static com.google.common.io.BaseEncoding.base64;
```

Comment:
> Serializing to a UTF string doesn't work for all possible values

just "a String"

"values" -> "codepoints" ?

Reply:
Technically it is still serialized into a String (that contains only base64 characters). By values I meant all possible values that a `Slice` can contain

Reply:
I don't know what "UTF string" is.

let's change commit message to sth like

```
Interpreting as UTF-8 works only for UTF-8 encoded text (and even then not necessarily for all the codepoints), 
but a Slice can carry arbitrary bytes.
```

Reply:
Sounds good, let me change the commit message